pages/linux/flock.md

```diff
@@ -0,0 +1,16 @@
+# flock
+
+> Manage locks from shell scripts.
+> You can use it to ensure that only one progress of your command is running.
+
+- Run a command with a file lock, wait if the lock is required by others:
+
+`flock /tmp/lock.lock -c "execute some command"`
```

Comment:
```suggestion
`flock {{/tmp/lock.lock}} --command "{{command}}"`
```
* We use the `{{...}}`-syntax for highlighting arguments. 
* We always use long options when possible.
* Does the file always have to be `/tmp/lock.lock`?

Reply:
The file path can be any path. command use the same file path share the same lock.